src/dsp_tools/utils/excel_to_json/properties.py

```diff
@@ -0,0 +1,457 @@
+import importlib.resources
+import json
+import warnings
+from typing import Any, Optional
+import jsonpath_ng.ext
+import jsonschema
+import regex
+import pandas as pd
+import numpy as np
+
+import dsp_tools.utils.excel_to_json.utils as utl
+from dsp_tools.models.exceptions import UserError
+
+languages = ["en", "de", "fr", "it", "rm"]
+language_label_col = ["label_en", "label_de", "label_fr", "label_it", "label_rm"]
+mandatory_properties = ["name", "object", "gui_element"]
+
+
+def _search_validation_error(
```

Comment:
If I read the function name, I don't understand what this function does. After reading the code, I understand that it transforms a JSON validation error into a user-friendly error message. So what do you think about something like `_prepare_err_msg_from_json_validation_error()` as function name?